native_client/coqui-stt.h

```diff
@@ -326,6 +361,20 @@ Metadata* STT_IntermediateDecodeWithMetadata(const StreamingState* aSctx,
  * @note This method will free the state pointer (@p aSctx).
  */
 STT_EXPORT
+char* STT_IntermediateDecodeExpensive(StreamingState* aSctx);
+
+/**
+ * @brief Compute the intermediate decoding of an ongoing streaming inference,
+ *
+ * @param aSctx A streaming state pointer returned by {@link STT_CreateStream()}.
+ * @param aNumResults The number of candidate transcripts to return.
+ *
+ * @return Metadata struct containing multiple candidate transcripts. Each transcript
+ *         has per-token metadata including timing information. The user is
+ *         responsible for freeing Metadata by calling {@link STT_FreeMetadata()}.
+ *         Returns NULL on error.
+ */
+STT_EXPORT
```

Comment:
Looks like somehow the merges reintroduced the problem of this comment being on the wrong function?